libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp

```diff
@@ -256,40 +256,42 @@ bool AP_ExternalAHRS_InertialLabs::check_uart()
         switch (mtype) {
         case MessageType::GPS_INS_TIME_MS: {
             // this is the GPS tow timestamp in ms for when the IMU data was sampled
-            CHECK_SIZE(u.gps_time_ms);
-            state2.gnss_ins_time_ms = u.gps_time_ms;
+            CHECK_SIZE(u.gnss_time_ms);
+            nav_ins_data.ms_tow = u.gnss_time_ms;
             break;
         }
         case MessageType::GPS_WEEK: {
-            CHECK_SIZE(u.gps_week);
-            gps_data.gps_week = u.gps_week;
+            CHECK_SIZE(u.gnss_week);
+            nav_ins_data.gps_week = u.gnss_week;
             break;
         }
         case MessageType::ACCEL_DATA_HR: {
             CHECK_SIZE(u.accel_data_hr);
-            ins_data.accel = u.accel_data_hr.tofloat().rfu_to_frd()*GRAVITY_MSS*1.0e-6;
+            ins_data.accel = u.accel_data_hr.tofloat().rfu_to_frd()*9.8106f*1.0e-6; // m/s^2
```

Comment:
Why are you replacing `GRAVITY_MSS` here?  It's the sort of thing that someone will come through at some stage and revert, so it definitely deserves a comment!

Reply:
Done, I added a comment about it